src/AasxServerBlazor/Pages/Db.razor

```diff
@@ -1,905 +1,803 @@
+                                    </td>
+                                    <td><a href="@link" target="_blank">@link</a></td>
+                                </tr>
+                            }
+                        </tbody>
+                    </table>
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">All other as text</span></b>
+                        <br />
+                        @for (int i = 1000; i < data.Count; i++)
+                        {
+                            var smDB = data[i];
+                            string submodel64 = Base64UrlEncoder.Encode((smDB.Identifier));
+                            string link = AasxServer.Program.externalBlazor + "/submodels/" + submodel64;
+                            int smeCount = db.SMESets.Where(sme => sme.SMId == smDB.Id).Count();
+                            <span style="background-color:blue;">&nbsp</span>
+                            <span>@smDB.AASXId</span>
+                            <span style="background-color:blue;">&nbsp</span>
+                            <span>@smDB.AASId</span>
+                            <span smDB="background-color:blue;">&nbsp</span>
+                            <span><b>@smDB.Id</b></span>
+                            <span smDB="background-color:blue;">&nbsp</span>
+                            <span>@smDB.Identifier</span>
+                            <span smDB="background-color:blue;">&nbsp</span>
+                            <span>@smDB.IdShort</span>
+                            <span smDB="background-color:blue;">&nbsp</span>
+                            <span>@smDB.SemanticId</span>
+                            <span style="background-color:blue;">&nbsp</span>
+                            <a href="@(shorturl + "/aasx?aasxid=" + smDB.AASXId)" target="_blank">aasx </a>
+                            <a href="@(shorturl + "/aas?aasid=" + smDB.AASId)" target="_blank">aas </a>
+                            <a href="@(shorturl + "/sme?smid=" + smDB.Id)" target="_blank">sme:@(smeCount)</a>
+                            <span style="background-color:blue;">&nbsp</span>
+                            <a href="@link" target="_blank">@link</a>
+                            <br />
+                        }
+                        <br />
+                    }
+                    <span>@("Count: " + data.Count)</span>
+                    <br />
+                }
+
+                if (url.EndsWith("/smjson") && smidentifier != "")
+                {
+                    var smDBList = db.SMSets.Where(s => s.Identifier.Equals(smidentifier)).ToList();
+                    if (smDBList.Any())
+                    {
+                        var smDB = smDBList.First();
+                        var submodel = ReadDB.GetSubmodel(smDB: smDB);
+
+                        string json =  "";

```

Comment:
```suggestion
                        string json =  string.Empty;

```

please use string.Empty directly. Otherwise someone might add a space here and this results in an error somehow